harmonisation/specimen/basisOfRecord.md

```diff
@@ -0,0 +1,29 @@
+# Basis of Record
+
+## openDS
+Will be harmonised to `dwc:basisOfRecord`
+
+In general this field cannot be empty, as we use it to determine whether we should process the specimen or not.
+Specimen with Basis of Record that are accepted are:
+- Preserved Specimen
+- Fossil Specimen
+- Material Sample
+- Living Specimen
+- Rock Specimen
+- Mineral Specimen
+- Meteorite specimen
```

Comment:
There is talk about adding the Soil category to GeoCASe, might be interesting to follow this development and add the category if it gets through

Reply:
Not sure if it is ok to add rock, mineral and meteorite specimen to BasisOfRecord as this is not compatible with the DwC recommendation to refer to an existing DwC class. Should it not be left empty in those cases?